### PR TITLE
Update travis config

### PR DESCRIPTION
**Pull request by mischah**
* Add npm lockfiles
* Add latest node version to travis config

### CODE REVIEW

**Comment by mischah** [comment]
One test is failing on travis. Locally all tests are passing 🤔 

```
  1) Environment Resolver #lookup() register global generators:
     AssertionError [ERR_ASSERTION]: undefined == true
      at Context.<anonymous> (test/resolver.js:78:14)
      at Environment.resolver.lookup (lib/resolver.js:19:723)
      at Context.<anonymous> (test/resolver.js:53:16)
```

File: test/resolver.js [hunk 1]
```diff
@@ -75,6 +75,8 @@ describe('Environment Resolver', function () {
     });
 
     globalLookupTest('register global generators', function () {
+      console.log('hej');
+      console.log(this.env.get('dummytest:app'));
```
Comment (mischah) [inline test/resolver.js]: `this.env.get('dummytest:app')` is `undefined` on travis. 

… and I have no clue why 😕

**Comment by mischah** [comment]
I could need some help over here. I’m totally clueless.

File: test/resolver.js (hunk 1 above)
Comment (SBoudrias) [reply test/resolver.js]: We install generator dummytest globally for this test https://github.com/yeoman/generator-dummytest.

Most likely something changed around how this is handled.

**Comment by SBoudrias** [comment]
Took me quite a while to fix... but finally got around to fix them it on master.

There was an issue with duplicated paths and wrong ordering of resolution I believe.

**Pull request closed without merging**